packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -374,7 +371,7 @@ export class GarbageCollector implements IGarbageCollector {
 			return;
```

Comment:
I think `initializeOrUpdateGCState` should early return if it's not enabled, to avoid all the logging on "connected"

Reply:
`initializeOrUpdateGCState` is called from `setConnectionState` which already has this check.